client/src/App.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useEffect, useState } from "react";
 import logo from "./logo.svg";
 import "./App.css";
 import Communication from "./components/Communication";
+import EpmployerAddForm from "./components/EmployerAddForm";
```

Comment:
```suggestion
import EmployerAddForm from "./components/EmployerAddForm";
```
Just a minor typo in this file and in the components file